### PR TITLE
[Chronological Tabs] Fix #6430: Add ability to add tabs

### LINKED ISSUE #6430: [Chronological Tabs] Add ability to add tabs
**Vision**

As a user, I want to open a new tab so that I can search for something new but keep my other tab open for reference

-----

**Acceptance Criteria**

* User can open a new tab (add button at bottom of tab tray [mock](https://app.abstract.com/share/860869c6-bb5a-4200-9fd5-cc6e59eecaf1?mode=design&sha=08f777d6dc8395bc9cdb30edfb809507e9eff870))
* A new tab can be spawned from an existing tab by opening a link in a new tab

**Not Included**

* This doesn't include group by date, so the tab can just be added to the bottom of the list

### PR DESCRIPTION
**Pull request by dnarcese**
- selecting a tab and adding a new tab goes to top of the list
- add tab button works
- delete all tab buttons works
- private tabs button _does not_ work yet

### CODE REVIEW

File: Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift [hunk 1]
```diff
@@ -66,12 +81,47 @@ extension TabTrayV2ViewController: UITableViewDataSource {
...
+    
+    @objc func didTapToolbarAddTab(_ sender: UIButton) {
+        viewModel.addTab()
+        dismissTabTray()
+    }
+    
+    @objc func didTapToolbarDelete(_ sender: UIButton) {
+        let controller = AlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        controller.addAction(UIAlertAction(title: Strings.AppMenuCloseAllTabsTitleString, style: .default, handler: { _ in self.viewModel.closeTabsForCurrentTray() }), accessibilityIdentifier: "TabTrayController.deleteButton.closeAll")
+        controller.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment: "Label for Cancel button"), style: .cancel, handler: nil), accessibilityIdentifier: "TabTrayController.deleteButton.cancel")
```
Comment (nbhasin2) [inline Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift:93]: nit: I am guessing this string is temporary and will be moved to Strings file

File: Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift [hunk 2]
```diff
@@ -66,12 +81,47 @@ extension TabTrayV2ViewController: UITableViewDataSource {
...
+        controller.popoverPresentationController?.sourceView = sender
+        controller.popoverPresentationController?.sourceRect = sender.bounds
+        present(controller, animated: true, completion: nil)
+    }
+    
+    func hideDisplayedTabs( completion: @escaping () -> Void) {
+           let cells = tableView.visibleCells
+
+           UIView.animate(withDuration: 0.2,
```
Comment (nbhasin2) [inline Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift:102]: Indentation seems really off, is it just github? 🤔

File: Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift (hunk 1 above)
Comment (dnarcese) [reply Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift:93]: Ya I should change that. This block is copied from the old implementation.

File: Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift (hunk 2 above)
Comment (dnarcese) [reply Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift:102]: This is the indentation copied from the old implementation, so I didn't change it.

**Pull request merged**